Sources/tinys3/AWSRequest.swift

```diff
@@ -185,7 +216,9 @@ extension AWSRequest {
     }
 
     var canonicalQueryString: String {
-        queryItems.asEscapedQueryString
+        queryItems
+            .sorted { $0.name < $1.name }
+            .asEscapedQueryString
```

Comment:
This fix was necessary otherwise the new `listMultipartUploadsRequest` failed to be HMAC-signed properly, and caused the request to be rejected.

This is because when constructing that `AWSRequest`, I pass `uploads` queryParam first and then `prefix`, while properly signing a request with AWS4 requires the query parameters to be sorted alphabetically. I could have inverted the order of the `query:` array in my `AWSRequest` to solve this, but figured I'd solve it at the source instead.